### PR TITLE
Dev mobilenet v2

### PR DESCRIPTION
**Pull request by Flowingsun007**
- 1.加载pretrain model：test3张测试图片可正确分类，train时，loss在0.27x；
- 2.从头训练时：loss可正常下降，精度在10epoch内可达到67.x %+

### CODE REVIEW

File: mobilenetv2/README.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+## Inference on Single Image
+
+```bash
+bash infer.sh
+```
+
+#### Util
+pytorch pretrained module to oneflow
+```python
+wget https://www.dropbox.com/s/47tyzpofuuyyv1b/mobilenetv2_1.0-f2a8633.pth.tar?dl=1
```
Comment (Ldpe2G) [inline mobilenetv2/README.md:30]: torch vision mobilenetv2 模型的官方链接是这个吗

**Comment by Ldpe2G** [comment]
https://github.com/pytorch/vision/blob/master/torchvision/models/mobilenetv2.py

最好是能对齐torch vision官方的这个实现

File: mobilenetv2/README.md (hunk 1 above)
Comment (BBuf) [inline mobilenetv2/README.md:30]: 这里如果是pytorch的版本大于0.4.1并且网络里面有bn层会报错，建议改成：

```python
parameters = torch.load("mobilenet_v3_small-047dcff4.pth")
new_parameters = dict()
for key,value in parameters.items():
     if "num_batches_tracked" not in key:
          val = value.detach().cpu().numpy()
          new_parameters[key] = val
mobilenetv3_module.load_state_dict(new_parameters)
flow.save(mobilenetv3_module.state_dict(), "mobilenetv3_oneflow_model")
```

类似

File: mobilenetv2/train.py [hunk 2]
```diff
@@ -0,0 +1,136 @@
+import oneflow.experimental as flow
+import torch
```
Comment (BBuf) [inline mobilenetv2/train.py:2]: oneflow的训练脚本需要把imprt torch删掉。

Comment (Flowingsun007) [reply mobilenetv2/train.py]: 好的

File: mobilenetv2/README.md (hunk 1 above)
Comment (Flowingsun007) [reply mobilenetv2/README.md]: 已修改

File: mobilenetv2/README.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+```bash
+bash infer.sh
+```
+
+#### Model
+We provide a converted pretrained model(from pytroch), you can get [here](https://oneflow-public.oss-cn-beijing.aliyuncs.com/model_zoo/cv/classification/mobilenetv2/mobilenetv2_oneflow_model.zip)
+Also, you can use following steps to convert it on your own:
+
+```python
+wget https://download.pytorch.org/models/mobilenet_v2-b0353104.pth
```
Comment (Ldpe2G) [inline mobilenetv2/README.md:32]: 这个不是python脚本的内容，应该另外用bash的格式来写

File: mobilenetv2/models/mobilenetv2.py [hunk 4]
```diff
@@ -0,0 +1,208 @@
...
+from typing import Callable, Any, Optional, List
+
+import oneflow.experimental as flow
+from oneflow.experimental import nn, Tensor
+
+__all__ = ['MobileNetV2', 'mobilenet_v2']
+
+# reference: https://github.com/pytorch/vision/blob/master/torchvision/models/mobilenetv2.py
+
+# model_urls = {
```
Comment (Ldpe2G) [inline mobilenetv2/models/mobilenetv2.py:11]: 这个删掉吧

File: mobilenetv2/models/mobilenetv2.py [hunk 5]
```diff
@@ -0,0 +1,208 @@
+import math
+from typing import Callable, Any, Optional, List
+
+import oneflow.experimental as flow
+from oneflow.experimental import nn, Tensor
+
+__all__ = ['MobileNetV2', 'mobilenet_v2']
+
+# reference: https://github.com/pytorch/vision/blob/master/torchvision/models/mobilenetv2.py
```
Comment (Ldpe2G) [inline mobilenetv2/models/mobilenetv2.py:9]: 下面205行已经写了地址了

File: mobilenetv2/README.md (hunk 3 above)
Comment (Flowingsun007) [reply mobilenetv2/README.md:32]: 好的

File: mobilenetv2/models/mobilenetv2.py (hunk 4 above)
Comment (Flowingsun007) [reply mobilenetv2/models/mobilenetv2.py]: 已删

File: mobilenetv2/models/mobilenetv2.py (hunk 5 above)
Comment (Flowingsun007) [reply mobilenetv2/models/mobilenetv2.py]: 已删

**Review by Ldpe2G — APPROVED** [review]
